index.html

```diff
@@ -0,0 +1,171 @@
+          <li class="nav-item home">
+            <a href="./index.html" class="nav-link"
+              ><i class="fas fa-home"></i
+            ></a>
+          </li>
+          <li class="nav-item">
+            <a href="./about.html" class="nav-link">About</a>
+          </li>
+          <li class="nav-item">
+            <a href="./about.html" class="nav-link">Program</a>
+          </li>
+          <li class="nav-item">
+            <a href="#Join" class="nav-link">Join</a>
+          </li>
+          <li class="nav-item">
+            <a href="#Sponsor" class="nav-link">Sponsor</a>
+          </li>
+          <li class="nav-item">
+            <a href="#News" class="nav-link">News</a>
+          </li>
+          <li class="nav-item">
+            <a href="#enroll" class="nav-link enroll">Enroll</a>
+          </li>
+         
+        </ul>
+        <div class="hamburger">
+            <span class="bar"></span>
+            <span class="bar"></span>
+            <span class="bar"></span>
+          </div>
+      </nav>
+    </header>
+    <section class="container container-sc">
+      <div class="headline">
+        <p class="welcome-msg">"Welcome to Our Digital Jerusalem"</p>
+        <h1>
+          CREATIVELY START YOUR <br />
+          CYBER SECURITY <br />
+          ACTIVITIES
+        </h1>
+      </div>
+      <div class="headline-box">
+        <p>
+          Cybersecurity is the protection of internet-connected systems such as
+          hardware, software and data from cyberthreats
+        </p>
+      </div>
+      <div class="headline-start">
+        <h2>2022.8.8 (MON) ~ 12 FRI</h2>
+        <p>
+          @Our Digital Jerusalem Company , Cyber Security activities and more
+        </p>
+      </div>
+    </section>
+    <section class="program">
+      <h2>Main Program</h2>
+      <hr class="hr" />
+      <div class="contanier-card">
+        <div class="program-card">
+          <img src="./images/icon_presenter.png" alt="" class="img-card" />
+          <h3 class="title-card">Lecture</h3>
+          <p class="text-card">
+            Listen to the speakers about Cyber Security and how to be safe by
+            using the internet
+          </p>
+        </div>
+        <div class="program-card">
+          <img src="./images/icon_program.png" alt="" class="img-card" />
+          <h3 class="title-card">Exhibition</h3>
+          <p class="text-card">
+            Lorem ipsum dolor sit amet consectetur adipisicing elit. Quibusdam,
+            animi?
+          </p>
+        </div>
+        <div class="program-card">
+          <img src="./images/icon_messages.png" alt="" class="img-card" />
+          <h3 class="title-card">Forum</h3>
+          <p class="text-card">
+            Have the time to share your thoughts and opinions with experts for
+            each topic
+          </p>
+        </div>
+        <div class="program-card">
+          <img src="./images/icon_board.png" alt="" class="img-card" />
```

Comment:
- [ ]  Kindly add alt texts to all images, please do this because if your image is not rendered due to network issues or a broken link, users can still see a text that says what the image about, this make your website more accessible  to users 👍🏿